### ISSUE TITLE
Handy Tips

### ISSUE THREAD

**Issue by joserochh** [Funnyach/MsgGeneration#4]
The script can't be run relative from any place, it only works when run from inside MsgGeneration/bin directory.  Otherwise it can't find config/BaseTypes.txt file. 
It would be useful also to be able to specify an output directory for the new headers. Even thought documentation says "the generated C++ files will be placed in the main folder of the package" it wasn't the case for me as the files ended up in ROS' src/ folder.

**Comment by Funnyach** [comment]
You are quite right about the first part. I have not updated the readme for a long time, I will make sure to include this.

I would also like to be able to specify an ouput directory and will be working on that. Might take a while though. I don't really understand how the files ended up in ROS' src/ folder. That never happened for me and should theoretically not be possible, as not a singe mention of that folder anywhere in the code. I will try to look into it though.
